src/components/pets/pets.css

```diff
@@ -49,3 +49,12 @@
   align-self: center;
   border-radius: 4;
 }
+.card{
```

Comment:
we can do 
```
petList__container .card {
//css here
}
```
remove the quoation marks and make sure to add px to the `padding-top : 10px`